### PR TITLE
For #877 added JUnit v5.

### PR DESCRIPTION
**Pull request by oridan**
For #877 added JUnit v5.

### CODE REVIEW

**Comment by 0crat** [comment]
@oridan[/z](https://www.0crat.com/u/oridan) this pull request is too small, just 6 lines changed (less than 10), there will be no formal code review, see [§53](http://www.zerocracy.com/policy.html#53) and [§28](http://www.zerocracy.com/policy.html#28); in the future, try to make sure your pull requests are not too small; @paulodamaso[/z](https://www.0crat.com/u/paulodamaso) please review this and merge or reject

**Comment by paulodamaso** [comment]
@0crat in

**Comment by 0crat** [comment]
> @0crat in ([here](https://github.com/yegor256/takes/issues/892#issuecomment-447060695))

@paulodamaso Job [#892](https://github.com/yegor256/takes/issues/892) is now in scope, role is `REV`

**Comment by oridan** [comment]
@paulodamaso please, run travis again.

**Comment by oridan** [comment]
@paulodamaso please, check codecov.

**Comment by paulodamaso** [comment]
@0crat assign me

**Comment by 0crat** [comment]
> @0crat assign me ([here](https://github.com/yegor256/takes/issues/892#issuecomment-447149756))

@paulodamaso @paulodamaso[/z](https://www.0crat.com/u/paulodamaso) not enough funds available in the project, can't set budget of job [#892](https://github.com/yegor256/takes/issues/892), see [§21](http://www.zerocracy.com/policy.html#21); @paulodamaso[/z](https://www.0crat.com/u/paulodamaso) will get no money on completion; in order to fix that, add funds to the project and assign the job again

**Comment by 0crat** [comment]
Manual assignment of issues is discouraged, see [§19](http://www.zerocracy.com/policy.html#19): -5 point(s) just awarded to @paulodamaso[/z](https://www.0crat.com/u/paulodamaso)

File: src/test/java/org/takes/facets/auth/codecs/CcStrictTest.java [hunk 1]
```diff
@@ -39,6 +39,11 @@
     /**
      * CcStrict can block empty urn.
      * @throws IOException If some problem inside
+     * @todo #877:30min replace all org.junit.Test occurrences in the project
+     *  by JUnit 5 equivalent org.junit.jupiter.api.Test and replaces
+     *  all expected Exceptions by it's lamba style. Check
```
Comment (paulodamaso) [inline src/test/java/org/takes/facets/auth/codecs/CcStrictTest.java]: @oridan Please add that we should use `Assertions.assertThrows` in assertions expecting exceptions and correct "lambda" typo

**Comment by paulodamaso** [comment]
@oridan Please take a look to my comment in https://github.com/yegor256/takes/pull/892#pullrequestreview-184905633

**Comment by paulodamaso** [comment]
@rultor merge

**Comment by 0crat** [comment]
Job [#892](https://github.com/yegor256/takes/issues/892) is not in the agenda of @paulodamaso[/z](https://www.0crat.com/u/paulodamaso), can't retrieve data and time of add

**Comment by 0crat** [comment]
Job was finished in 14 hours, bonus for fast delivery is possible (see [§36](http://www.zerocracy.com/policy.html#36))

**Comment by 0crat** [comment]
The job [#892](https://github.com/yegor256/takes/issues/892) is now out of scope

**Comment by 0crat** [comment]
Order was finished: +35 point(s) just awarded to @paulodamaso[/z](https://www.0crat.com/u/paulodamaso)

**Pull request merged**